package.json

```diff
@@ -34,7 +34,7 @@
     "@primer/octicons-react": "^19.8.0",
     "@seald-io/nedb": "^4.0.2",
     "axios": "^1.6.0",
-    "bcrypt": "^5.1.1",
+    "bcryptjs": "^2.4.3",
```

Comment:
According to npm, [`bcryptjs`](https://www.npmjs.com/package/bcryptjs?activeTab=versions) has a single collaborator and last publish a version 8 years ago, whereas [`bcrypt`](https://www.npmjs.com/package/bcrypt?activeTab=versions) has 5 collaborators and last published a version 1 year ago. Is maintenance a concern with this dependency?

Reply:
Thanks for the question @rgmz and for the clarification @06kellyjac. I'm happy to proceed with a merge, but will propose that we migrate our authentication services to a plugin-based model. For our usage of GitProxy, we do not require the `file` based password service (using `bcrypt`) however are trumped by installation issues associated with a child dependency and running of `node-gyp`.